tox.ini

```diff
@@ -57,18 +57,22 @@ deps =
     parameterized
     pytest
     coverage[toml]
-    -r{toxinidir}/requirements.txt
+    -r {tox_root}/requirements.txt
 commands =
     coverage run --source={[vars]src_path} \
-        -m pytest --ignore={[vars]tst_path}integration -v --tb native -s {posargs}
+        -m pytest -v --tb native -s {posargs} {[vars]tests_path}/unit
     coverage report
 
 [testenv:integration]
 description = Run integration tests
+pass_env =
+    {[testenv]pass_env}
+    CI
+    CI_PACKED_CHARMS
 deps =
     pytest
     juju==2.9.11
     pytest-operator
-    -r{toxinidir}/requirements.txt
+    -r {tox_root}/requirements.txt
 commands =
-    pytest -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs}
+    pytest -v --tb native --log-cli-level=INFO -s {posargs} {[vars]tests_path}/integration/test_s3_charm.py
```

Comment:
Could we just make it a bit more general? like just do  `{[vars]tests_path}/integration`

I mean, I see the use case of someone adding a integration tests and then not running it

Reply:
Not really: [here](https://github.com/canonical/zookeeper-operator/blob/feature/cache-charmcraft-pack/tox.ini#L94) for instance

Reply:
I believe it makes sense that the `[testenv:integration]` runs all integrations, other more specific like `[testenv:integration-charm]` run the specific file.

Reply:
I understand the scope of the PR, although in this specific case is not producing much improvements since it has only one file. But I believe it HAS value to already have the structure in place for being efficient should we add more files, hence the PR. 

Here I'm just saying that it is consistent with other repo to call `[testenv:integration]` to run all integration environments, that's why I would just remove the filename. 

When one creates other files and wants to speed up things, they will create other envs called `[testenv:integratoin-{suffix}]` and run specific file on them `test_{suffix}.py`.